### PR TITLE
 Add: Info Button Tooltip

### PR DESCRIPTION
**Pull request by shadabkhan16**
## Description
Add an info button in the airdrop's incentive component. 

- What does this PR do?
- Why is it needed?

## Screenshots
Screenshot 2024-12-05 at 4 01 04 PM

### Desktop:
[Add screenshot here]

### Mobile:
[Add screenshot here]

## Approach
Add div tags  flex

- How did you solve the problem?
- Any particular design patterns or optimizations?

## Issue Number (if any)
Link to the issue this PR addresses, if applicable:

Fixes #[issue_number]

## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx [hunk 1]
```diff
@@ -36,14 +36,28 @@ const INCENTIVES_DETAILS = [
...
+                        <img src='/assets/images/ufo.svg' alt='ufo' width={20} height={20} loading='lazy'/>
+                        <p>{points.toString()}</p>
+                        
+                    </div>
+            
+                  </div>
                 </div>
             </div>
+            <div className='incentive-info-svg'>
+                <img src="/assets/images/Info.svg" alt="" />
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx:55]: Add alt attribute as well as `loading='lazy'` attribute

File: src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx [hunk 2]
```diff
@@ -36,14 +36,28 @@ const INCENTIVES_DETAILS = [
...
+                    <div className='incentive-card-type-points'>
+                        <img src='/assets/images/ufo.svg' alt='ufo' width={20} height={20} loading='lazy'/>
+                        <p>{points.toString()}</p>
+                        
+                    </div>
+            
+                  </div>
                 </div>
             </div>
+            <div className='incentive-info-svg'>
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx:54]: Separate this into a different component
Example : 
```jsx
<InfoToolTip/>
```

**Review by ArjunQBTech — CHANGES_REQUESTED** [review]

File: src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx (hunk 1 above)
Comment (shadabkhan16) [reply src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx]: Screenshot 2024-12-06 at 10 11 01 AM
All necessary changes have been made.

File: src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx [hunk 3]
```diff
@@ -36,14 +37,29 @@ const INCENTIVES_DETAILS = [
...
+             
+                <InfoTooltip title={type}> <img src="/assets/images/Info.svg" alt="info" loading='lazy' width={20} height={20} /> </InfoTooltip>
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx:57]: Add the children to next line

File: src/game_frontend/src/components/ui/Tooltip/index.tsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+interface InfoTooltipProps {
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/ui/Tooltip/index.tsx:3]: Rename to Tooltip

File: src/game_frontend/src/components/ui/Tooltip/index.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  title: string; // Tooltip content as a  string 
+  children: React.ReactNode; // The element triggering the tooltip
+}
+
+const InfoTooltip: React.FC<InfoTooltipProps> = ({ title, children }) => {
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/ui/Tooltip/index.tsx:8]: Rename to Tooltip

**Review by ArjunQBTech — CHANGES_REQUESTED** [review]

File: src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx [hunk 6]
```diff
@@ -36,14 +37,25 @@ const INCENTIVES_DETAILS = [
...
+                    <div className='incentive-card-type-points'>
+                        <img src='/assets/images/ufo.svg' alt='ufo' width={20} height={20} loading='lazy'/>
+                        <p>{points.toString()}</p>
+                        
+                    </div>
+                  </div>
                 </div>
             </div>
+            <div className='incentive-info-svg'>
+                <Tooltip title={type}>
```
Comment (ArjunQBTech) [inline src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx:55]: Create a separate component : 
```tsx
const InfoToolTip=()=>{
       return (
        <Tooltip title={type}>
                     <img src="/assets/images/Info.svg" alt="info" loading='lazy' width={20} height={20} />
                </Tooltip>
       )
}
```

**Review by ArjunQBTech — CHANGES_REQUESTED** [review]

File: src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx (hunk 6 above)
Comment (shadabkhan16) [reply src/game_frontend/src/components/dashboard/airdrop/Incentives/index.tsx]: separate component for InfoToolTip

**Review by ArjunQBTech — APPROVED** [review]
Great 👍

**Pull request merged**